src/backend/base/langflow/schema/message.py

```diff
@@ -171,20 +171,23 @@ async def get_file_content_dicts(self):
     def load_lc_prompt(self):
```

Comment:
what about adding an unit test ? 
we can easily spot this regression again with it